### PR TITLE
API-7907: allign changes

### PR DESCRIPTION
**Pull request by czeczot**

### CODE REVIEW

File: configuration/internal.go [hunk 1]
```diff
@@ -38,20 +38,18 @@ type updateBotRequest struct {
...
 }
 
-type listBotsResponse struct {
-	BotAgents []*BotAgent `json:"bot_agents"`
-}
+type listBotsResponse []*Bot
 
 type getBotRequest struct {
-	BotID string `json:"bot_agent_id"`
+	BotID  string   `json:"bot_agent_id"`
```
Comment (lpmichalik) [inline configuration/internal.go:48]: json:id ?

**Review by lpmichalik — APPROVED** [review]

File: configuration/internal.go (hunk 1 above)
Comment (czeczot) [reply configuration/internal.go]: done

**Comment by czeczot** [comment]
https://livechatinc.atlassian.net/browse/API-7905

File: configuration/structures.go [hunk 2]
```diff
@@ -65,24 +65,21 @@ func NewChatMemberIDsFilter(agents []string, inclusive bool) *chatMemberIDsFilte
...
-	} `json:"application"`
-	MaxChatsCount uint           `json:"max_chats_count"`
-	Groups        []*GroupConfig `json:"groups"`
-	Webhooks      *BotWebhooks   `json:"webhooks"`
+// Bot represents basic bot agent information
+type Bot struct {
+	ID     string `json:"id"`
+	Name   string `json:"name"`
+	Avatar string `json:"avatar_path"`
+}
```
Comment (janowsiany) [inline configuration/structures.go:73]: What about `work_scheduler`,  `groups`, `job_title`, `mobile`, `max_chats_count`, `owner_client_id `

File: configuration/structures.go [hunk 3]
```diff
@@ -65,24 +65,21 @@ func NewChatMemberIDsFilter(agents []string, inclusive bool) *chatMemberIDsFilte
...
+
+// BotDetails represents detailed bot agent information
+type BotDetails struct {
+	Bot
+	DefaultGroupPriority GroupPriority  `json:"default_group_priority"`
+	ClientID             string         `json:"owner_client_id"`
+	MaxChatsCount        uint           `json:"max_chats_count"`
+	Groups               []*GroupConfig `json:"groups"`
+	Webhooks             *BotWebhooks   `json:"webhooks"`
```
Comment (janowsiany) [inline configuration/structures.go:82]: What about `work_scheduler`, `job_title`, `mobile`

File: configuration/structures.go [hunk 4]
```diff
@@ -65,24 +65,21 @@ func NewChatMemberIDsFilter(agents []string, inclusive bool) *chatMemberIDsFilte
...
-		ClientID string `json:"client_id"`
-	} `json:"application"`
-	MaxChatsCount uint           `json:"max_chats_count"`
-	Groups        []*GroupConfig `json:"groups"`
-	Webhooks      *BotWebhooks   `json:"webhooks"`
+// Bot represents basic bot agent information
+type Bot struct {
+	ID     string `json:"id"`
+	Name   string `json:"name"`
+	Avatar string `json:"avatar_path"`
```
Comment (janowsiany) [inline configuration/structures.go:72]: In Agent structure this field is called AvatarPath, shouldn't we align it as well?

**Review by janowsiany — CHANGES_REQUESTED** [review]

**Review by janowsiany — CHANGES_REQUESTED** [review]
Generally, i am not totally sure but the difference get_bot(BotDetails) and list_bots(Bot) responses is webhooks field which is only available in get_bot method. Could you please confirm that?

**Comment by czeczot** [comment]
@janowsiany  Additional fields in get_bot response are: DefaultGroupPriority, ClientID and Webhooks.

**Review by janowsiany — APPROVED** [review]

**Review by janowsiany — APPROVED** [review]

**Review by kogierman — APPROVED** [review]

**Review by janowsiany — APPROVED** [review]

**Pull request merged**